electron/app/index.tsx

```diff
@@ -3,9 +3,11 @@ import { render } from 'react-dom';
 import { AppContainer as ReactHotAppContainer } from 'react-hot-loader';
 import './app.global.css';
 import * as sqlite3 from 'sqlite3';
-import log from 'electron-log';
```

Comment:
why the removal of electron-log here? Oh is it in order for you to more easily debug in the window?

Reply:
We should use electron-log for everything. Including things we want to see in the console. See: https://www.npmjs.com/package/electron-log#transport